### PR TITLE
Edits to Discussion

### LINKED ISSUE #185: Fix transition to paragraph 4
The edits in towards the start of the discussion in https://github.com/weecology/bbs-forecasting/commit/6185793257a0e161d33907d9d40918b6f9778930 mean that transition to paragraph 4 no longer makes any sense.

### LINKED ISSUE #186: Transitioning to conclusions
With all of the material added to the discussion the conclusions paragraph returns someone disconnected. This may be unavoidable, but it's quite abrupt at the moment.

### PR DESCRIPTION
**Pull request by ethanwhite**
* Rework first 4 paragraphs back into 3 for better flow
* Combine two time-scale paragraphs and reduce emphasis on observed convergence
* Remove paragraph level headers and improve overall structure and flow instead
* Add reference for not being able to address imperfect detection
* General editorial work throughout

Fixes #185. Fixes #186.

### CODE REVIEW

**Comment by ethanwhite** [comment]
This is definitely still a WIP, but you can see the direction I'm heading in case anyone has any objections. Basically I'm just trying to reintroduce the logical flow of the Discussion, which had gotten lost in a few places through an accumulation of paragraph level edits and the addition of new paragraphs. The big thing I'm waiting on now is finding out whether the model fit convergence results held up in the last analysis run. Once I know that I can proceed with finishing the edits to the Discussion.

**Comment by ethanwhite** [comment]
OK, here's the complete PR. Sorry for the big changes late. Let me know if there are questions and what things you'd both like to see changed. That can include putting things back that I pulled or reduced. My emphasis was on flow, but if that took away from important specifics we'll figure out how to get them back in.

**Comment by davharris** [comment]
Thanks Ethan!

File: manuscript/draft.Rmd [hunk 1]
```diff
@@ -507,219 +507,198 @@ fit.](../figures/observers.png)
...
+into the future, so it is important to evaluate forecasts at different lead
+times [Box 1; @petchey2015]. For deviance (the error metric that evaluates both
+models' point estimates and their uncertainty simultaneously), there is some
+indication that the JSDM and the nonlinear richness model converge towards the
+time-series and baseline models as the lead time of the forecast increases. This
+potential convergence is biologically plausible because at short lead times,
+environmental changes are relatively small and changes in the biota may lag
+behind changes in the environment. However, as the lead time of the forecast
+increases the amount of environmental change should increase and the systems
+will have more time to respond, potentially resulting in shifts towards improved
```
Comment (sdtaylor) [inline manuscript/draft.Rmd]: We should probably cite a recent Blonder paper here. It goes into all the details of the lag hypothesis. 

```
@article{blonder2017,
author = {Blonder, Benjamin and Moulton, Derek E. and Blois, Jessica and Enquist, Brian J. and Graae, Bente J. and Macias-Fauria, Marc and McGill, Brian and Nogu{\'{e}}, Sandra and Ordonez, Alejandro and Sandel, Brody and Svenning, Jens-Christian},
doi = {10.1111/ele.12736},
journal = {Ecology Letters},
month = {mar},
number = {3},
pages = {293--306},
title = {{Predictability in community dynamics}},
url = {http://doi.wiley.com/10.1111/ele.12736},
volume = {20},
year = {2017}
}
```

**Comment by sdtaylor** [comment]
:+1:

File: manuscript/draft.Rmd (hunk 1 above)
Comment (ethanwhite) [reply manuscript/draft.Rmd]: Thanks. Done.

Comment (ethanwhite) [reply manuscript/draft.Rmd]: Also added the Blois citation.

**Comment by davharris** [comment]
Okay, this looks like a big improvement overall.  As I read through, I noticed a few things that a) got lost in the edits or b) weren't very good in the first place.  But I think the easiest thing is probably to merge this and then make my own edits on top of it. Thanks @ethanwhite!

**Pull request merged**